packages/core/src/config/minimal.ts

```diff
@@ -19,6 +19,7 @@ const minimal: PluginStyleguideConfig<'built-in'> = {
     'no-enum-type-mismatch': 'warn',
     'no-unresolved-refs': 'error',
     'no-required-schema-properties-undefined': 'off',
+    'no-schema-type-mismatch': 'warn',
```

Comment:
I believe it's better not to turn it on for the `minimal` ruleset.

```suggestion
    'no-schema-type-mismatch': 'off',
```